legacy/nosql/mongo/test/mongotest.go

```diff
@@ -33,21 +36,29 @@ func MongoVersion(vers string) nosqltest.Database {
 				t.Fatal(err)
 			}
 
-			addr := cont.GetHostPort("27017/tcp")
-
+			addr := fmt.Sprintf("mongodb://%s", cont.GetHostPort("27017/tcp"))
 			err = pool.Retry(func() error {
-				sess, err := mgo.Dial(addr)
+				sess, err := gomongo.NewClient(options.Client().ApplyURI(addr))
+
+				if err != nil {
+					t.Fatal(err)
+				}
+
+				if err == nil {
```

Comment:
`if` can be dropped, assuming suggestion above

Reply:
updated.